### ISSUE TITLE
Re-Design Jekyll-themes

### ISSUE THREAD

**Issue by RizRiyz** [ariestiyansyah/jekyllthemes#1]
Fresh design and responsive needed. Plan to use bootstrap.

**Comment by jacobtomlinson** [comment]
:+1: It should compliment the jekyll website but also be noticeably different as it is not officially part of the project.

**Comment by jacobtomlinson** [comment]
We should also remove the adverts as raised in https://github.com/mattvh/jekyllthemes/issues/34

**Comment by jacobtomlinson** [comment]
Whoops just seen 849c09f8556f7693e712f3f3731301927de8f27e

**Comment by RizRiyz** [comment]
Yeah I removed the ads. now what do you think about bootstrap design? have idea on that?

**Comment by jacobtomlinson** [comment]
Bootstrap is a good choice. I think we should try and be fairly minimal and go with a flat design. Dark tones would be good as that's what Jekyll uses for their website.

**Comment by jacobtomlinson** [comment]
I would like to give more options to showcase designs if people want to. Perhaps optional extra screenshots which open larger in a lightbox or something.

**Comment by RizRiyz** [comment]
Ok I agree, you can handle that?

**Comment by jacobtomlinson** [comment]
Sure no problem. I'll start work on it and then open a PR for you to review.

**Comment by ddbullfrog** [comment]
bootstrap is awesome!

**Comment by Gaohaoyang** [comment]
@jacobtomlinson  I expect to see that!

**Comment by ddbullfrog** [comment]
@ariestiyansyah don't forget the coupon if you want to buy the domain on godaddy.
http://www.fatwallet.com/GoDaddy-coupons
